tests/test_lock.py

```diff
@@ -0,0 +1,220 @@
+import os
+import shutil
+import time
+
+import pandas as pd
+import pytest
+
+import audformat.testing
+import audeer
+import audiofile
+
+import audb
+
+
+os.environ['AUDB_CACHE_ROOT'] = pytest.CACHE_ROOT
+os.environ['AUDB_SHARED_CACHE_ROOT'] = pytest.SHARED_CACHE_ROOT
+audb.config.REPOSITORIES = pytest.REPOSITORIES
+
+
+DB_NAME = f'test_lock-{pytest.ID}'
+DB_ROOT = os.path.join(pytest.ROOT, 'db')
+DB_VERSION = '1.0.0'
+
+
+def clear_root(root: str):
+    root = audeer.path(root)
+    if os.path.exists(root):
+        shutil.rmtree(root)
```

Comment:
As we depend on `audeer>=1.18.0`, you can simply do:

```suggestion
    audeer.rmdir(root)
```